src/sentry/migrations/0377_fix_org_slug_casing.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.28 on 2023-01-31 20:37
+
+from django.db import IntegrityError, migrations
+from django.db.models.functions import Lower
+
+from sentry.constants import RESERVED_ORGANIZATION_SLUGS
+from sentry.db.models.utils import slugify_instance
```

Comment:
I know we generally try to avoid imports in migrations but these symbols have been stable for a long time.

Reply:
Had a look at the function, looks safe to import. It doesn't use any of our custom methods